JupyterLab/AppDelegate.swift

```diff
@@ -27,16 +27,22 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         guard let currentViewController = NSApp.mainWindow?.contentViewController as? ViewController else {
             return
         }
-        //let out : String = runSynchronousShellWithUserConfig(cmd: "open -a Terminal " + currentViewController.directory) ?? ""
-        _ = runSynchronousShell(launchPath: "/usr/bin/open", args: "-a", "Terminal", currentViewController.directory.replacingOccurrences(of: "\\ ", with: " ")) ?? ""
+        let terminalUrl = NSWorkspace.shared.urlForApplication(withBundleIdentifier: "com.googlecode.iterm2") ??
```

Comment:
While you are right that this is a more convenient way for opening, I think it might be nicer if this was configurable:
e.g. a setting pane with a custom Terminal Emulator to open in. For example: I have iTerm installed on my machine, but mainly use alacritty and Terminal and would be bummed if this now only opened in iTerm. It is not critical, just nice to have.

Reply:
That's a good idea. I solved this today by adding an NSPathControll to the settings. Unfortunately I found no way to easily change the popup menu so that it contains other/recently used terminal applications. Therefore it simply shows the parent folders with isn't really that helpful:

<img width="204" alt="Screenshot 2021-09-07 at 15 11 41" src="https://user-images.githubusercontent.com/29376715/132350723-57355ef7-657b-4c4b-a5ed-77d5a8b7e780.png">

Reply:
This is really nice!